projects/plugins/jetpack/modules/publicize/publicize-jetpack.php

```diff
@@ -174,33 +241,60 @@ function get_all_connections_for_user() {
 		return false;
 	}
 
-	function get_connection_id( $connection ) {
+	/**
+	 * Get the ID of a connection.
+	 *
+	 * @param array $connection The connection.
+	 * @return string
+	 */
+	public function get_connection_id( $connection ) {
 		return $connection['connection_data']['id'];
 	}
 
-	function get_connection_unique_id( $connection ) {
+	/**
+	 * Get the unique ID of a connection.
+	 *
+	 * @param array $connection The connection.
+	 * @return string
+	 */
+	public function get_connection_unique_id( $connection ) {
 		return $connection['connection_data']['token_id'];
 	}
 
-	function get_connection_meta( $connection ) {
-		$connection['user_id'] = $connection['connection_data']['user_id']; // Allows for shared connections
+	/**
+	 * Get the meta of a connection.
+	 *
+	 * @param array $connection The connection.
+	 * @return array
+	 */
+	public function get_connection_meta( $connection ) {
+		$connection['user_id'] = $connection['connection_data']['user_id']; // Allows for shared connections.
 		return $connection;
 	}
 
-	function admin_page_load() {
-		if ( isset( $_GET['action'] ) && 'error' === $_GET['action'] ) {
-					add_action( 'pre_admin_screen_sharing', array( $this, 'display_connection_error' ), 9 );
+	/**
+	 * Show error on settings page if applicable.
+	 */
+	public function admin_page_load() {
+		if ( filter_input( INPUT_GET, 'action', FILTER_SANITIZE_STRING ) === 'error' ) {
+			add_action( 'pre_admin_screen_sharing', array( $this, 'display_connection_error' ), 9 );
 		}
 	}
 
-	function display_connection_error() {
-		$code = false;
-		if ( isset( $_GET['service'] ) ) {
-			$service_name = $_GET['service'];
-			$error        = sprintf( __( 'There was a problem connecting to %s to create an authorized connection. Please try again in a moment.', 'jetpack' ), Publicize::get_service_label( $service_name ) );
+	/**
+	 * Display an error message.
+	 */
+	public function display_connection_error() {
+		$code            = false;
+		$service         = filter_input( INPUT_GET, 'service', FILTER_SANITIZE_STRING );
```

Comment:
I'm not sure what the preferred way of sanitizing GET parameters is, but it looks like we're using `sanitize_text_field` and `wp_unslash` in [some other places](https://github.com/Automattic/jetpack/blob/ba2dd6c30be137baad96aafb2466d0aed7e182e5/projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php#L81). Might be worth checking for consistency

Reply:
I personally like `filter_input` but I think consistency is more important here, I'll change it to use `sanitize_text_field` and `wp_unslash` instead 👍🏼

Reply:
Looking more closely this actually seems to be dead code, so it might not be worth changing anything here.

Reply:
Fair enough, I think you're using `filter_input` in a couple of other places in this PR, so it could be worth updating those.

Reply:
Removed the other instances in 76bcac4